pom.xml

```diff
@@ -1676,6 +1577,12 @@
                                     <version>${maven.min.version}</version>
                                 </requireMavenVersion>
                                 <banDuplicatePomDependencyVersions />
+                                <bannedDependencies>
+                                    <excludes>
+                                        <!-- We use Jandex 3 instead: io.smallrye:jandex, io.smallrye:jandex-maven-plugin -->
+                                        <exclude>org.jboss:jandex*</exclude>
+                                    </excludes>
+                                </bannedDependencies>
```

Comment:
I wonder whether we should also add rules for `javax` dependencies... at the very least the most important ones, such as JPA, CDI, transactions, etc. WDYT?